fpdf/output.py

```diff
@@ -974,6 +1017,18 @@ def _finalize_catalog(
             catalog_obj.names = pdf_dict(
                 {"/EmbeddedFiles": pdf_dict({"/Names": pdf_list(file_spec_names)})}
             )
+        page_labels = [
+            f"{seq} {pdf_dict(page[1].get_page_label().serialize())}"
+            for (seq, page) in enumerate(fpdf.pages.items())
+            if page[1].get_page_label()
+        ]
+        if page_labels and not fpdf.pages[1].get_page_label():
```

Comment:
Why considering only the second page (`pdf.pages[1]`) and not the 1st one? (`pdf.pages[0]`)

Reply:
fpdf.pages is a dictionary where the page number is the key. pdf.pages[1] is the first page of the document and 0 should never exist.